modules/lead/product-aws/aws.tf

```diff
@@ -19,7 +19,7 @@ resource "aws_codebuild_project" "codebuild_build" {
 
   source {
     type            = var.source_type
-    location        = each.value.sourcelocation
+    location        = "https://git-codecommit.us-east-1.amazonaws.com/v1/repos/${each.value.repo}"
```

Comment:
the source for codebuild should be codepipeline s3 bucket

Reply:
The codebuild source seems to get overwritten by the source of codepipeline so this still works

Reply:
Also that might create somewhat of a circular dependency in that the codebuild resource would need to wait for the execution of codepipeline before it had a valid source, but codepipeline can't run until the codebuild resource is created. Not sure if that would actually stop the creation of these resources though, I can try testing.

Reply:
There isn't a circular dependency.  You just set the codebuild source.type to CODEPIPELINE and location to the S3 bucket.

Reply:
Gotcha I thought you meant set the source.type to S3